CSS/style.css

```diff
@@ -270,8 +277,8 @@ h2 {
 .about-myself {
   display: flex;
   flex-direction: column;
-  padding: 114px 24px;
-  gap: 24px;
+  padding: 114px 7px 114px 90px;
+  gap: 0;
   background-color: white;
   border-top-right-radius: 80px;
 }
```

Comment:
Kindly update the styling of your about myself section to match the figma design better🙏

Your design:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/37116322/230654507-2a900ec7-d8fd-4483-bce3-d6babb4f8886.png">

Figma design:
<img width="529" alt="image" src="https://user-images.githubusercontent.com/37116322/230654540-329b3d51-e2dc-43be-8a82-f28d1f6b3fc3.png">

A few things to note:
- Kindly make sure the arrow icons on the right are aligned
- Kindly update the images and text in the languages dropdown to be left aligned not center aligned